test/test_helper.rb

```diff
@@ -1,6 +1,10 @@
-# See https://coveralls.io/docs/ruby
-require "coveralls"
-Coveralls.wear!("rails")
+require 'simplecov'
+SimpleCov.start
+
+if ENV['CI']=='true'
+  require 'codecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.